src/Features/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -52,12 +52,12 @@ Microsoft.CodeAnalysis.Completion.CompletionItemRules.CommitCharacterRules.get -
 Microsoft.CodeAnalysis.Completion.CompletionItemRules.EnterKeyRule.get -> Microsoft.CodeAnalysis.Completion.EnterKeyRule
 Microsoft.CodeAnalysis.Completion.CompletionItemRules.FilterCharacterRules.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.Completion.CharacterSetModificationRule>
 Microsoft.CodeAnalysis.Completion.CompletionItemRules.FormatOnCommit.get -> bool
-Microsoft.CodeAnalysis.Completion.CompletionItemRules.Preselect.get -> bool
+Microsoft.CodeAnalysis.Completion.CompletionItemRules.MatchPriority.get -> int
```

Comment:
Is this also unshipped in Update 3 as a part of the public APIs there? Does this need to be targeting stabilization?